src/js/api/getApi.js

```diff
@@ -0,0 +1,26 @@
+import Time from '../helpers/convertTime';
+
+const getWeatherData = async (city) => {
+  const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=b65d7b54d34b0aecf82c86c39aabf664&units=metric`);
+  const data = await response.json();
```

Comment:
**REQUIRED**
Error handling for `promises` or `async/await` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
 
It's not a good idea to expose API keys to the version control. The README file should contain instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them(if necessary).